curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-functional-programming-by-building-a-spreadsheet/646d44da986f2bc9b72f5fe2.md

```diff
@@ -7,7 +7,7 @@ dashedName: step-81
 
 # --description--
 
-Declare a `toNumberList` function which takes an `args` parameter, and returns the result of splitting the `args` by commas, and mapping the resulting array to `parseFloat`.
+Declare a `toNumberList` function that takes an `args` parameter and implicitly returns the result of splitting the args by commas. Then chain a `map` method to your `split` method and pass in `parseFloat` as a reference to the `map` method.
```

Comment:
```suggestion
Declare a `toNumberList` function that takes an `args` parameter and implicitly returns the result of splitting the args by commas. Then chain a `map` method to your `split` method and pass in `parseFloat` as the callback to the `map` method.
```

I think 

> pass in `parseFloat` as a reference to the `map` method.

sounds odd.

Reply:
This part was the initial problem, because were passing `parseFloat`  as a callback to the `map` and that's failing the tests. The just need to pass `parseFloat` as is, so I think we might need to be careful with the wording there.

Reply:
The failing example code in the issue is not passing `parseFloat` as the callback. It is passing an anonymous function.

I realize the use of "reference" here is supposed to clarify that you do not need a function wrapper but I think the sentence structure as it is right now is ambiguous.

Passing a function as the callback does not in any way imply wrapping it inside an anonymous function. If we haven't made that clear by now, then it is understandable that the camper might think you have to wrap the function inside an anonymous function. But at this point in the curriculum that really should not be the case.

By now, it should be very clear to the camper that the following ways of using `map` are equivalent.

```
const multiple = (number) => number * number;
console.log([2, 3, 4, 8].map(multiple)); // [4, 9, 16, 64]
console.log([2, 3, 4, 8].map((n) => multiple(n))); // [4, 9, 16, 64]
```

---

As an aside, I think unless we are teaching the concise syntax, and therefore requiring the concise syntax, both versions should pass as the result is the same.